lib/output.js

```diff
@@ -325,6 +329,36 @@ function tiff (options) {
       throw new Error('Invalid Value for squash ' + options.squash + ' Only Boolean Values allowed for options.squash.');
     }
   }
+  // tiling
+  if (is.object(options) && is.defined(options.tile)) {
```

Comment:
This might be the right time to merge all the logic that depends on the `if (is.object(options)) { ... }` statement being true. If nothing else it should reduce the complexity of this function.

Reply:
Merged all that logic together, squashed my commits, and rebased on master.